README.md

```diff
@@ -7,7 +7,7 @@
 
 A collection of lints to catch common mistakes and improve your [Rust](https://github.com/rust-lang/rust) code.
 
-[There are 259 lints included in this crate!](https://rust-lang-nursery.github.io/rust-clippy/master/index.html)
+[There are 260 lints included in this crate!](https://rust-lang-nursery.github.io/rust-clippy/master/index.html)
```

Comment:
I guess this (and some other stuff) got changed by the `update_lints.py` script. That's ok for now, but it could lead to some annoying rebasing on your side, if someone else changes this too. Consider to only add things in your PR, which is important for your lint/fix/... and make an additional PR for the `update_lints.py` script. It may also save you some work :wink: